internal/app/kdk/up.go

```diff
@@ -27,6 +29,12 @@ func Up(
 	dockerClient *client.Client,
 	cfg *kdkConfig,
 	logger logrus.Entry) error {
+	if runtime.GOOS == "windows" {
```

Comment:
MUST: I don't see the corresponding destroy logic.

Reply:
the mirror is shared between all kdk containers.  So there isn't really a clean/safe way to stop the mirror in the `destroy` command.  
We could add a `--all` flag to destroy or maybe add a `halt` command with `--all` flag to stop all running kdk containers and stop the mirror.
Thoughts?